### ISSUE TITLE
Question about momentum and eps in BatchNorm2d

### ISSUE THREAD

**Issue by djiajunustc** [traveller59/second.pytorch#364]
Hi,
I found that the momentum is set as 0.01 and eps is set as 1e-3, which is really different compared to the default setting of BN.
Since the eps is taken to avoid divisor being zero and causing nan, why should we take such a large number compared to the default setting 1e-5 ? Is this setting related to some property of pseudo images/pillars?
Besides, have you tried to replace the BN with GN?

Thanks a lot !
